apps/frontend/app/signup/fields.ts

```
@@ -0,0 +1,143 @@
+
+/*
+    Required fields:
+    email, password, first & last name, gender, branch, phoneNumber
+
+    Optional:
+    address fields (since there's different types of military addresses)
+    Address format based on https://support.govx.com/article?id=1176&sectionId=1025
+*/
+
+const emailPattern = new RegExp("^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$");
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)